### PR TITLE
Obriens' solution to mini-challenge-3

### PR DESCRIPTION
**Pull request by obrienu**
All tests passed
kindly review @bondz

### CODE REVIEW

File: src/binary-reversal/index.js [hunk 1]
```diff
@@ -3,6 +3,20 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function convertToReveredBinary(value) {
+    if (value <= 0) {
+        return "";
+    }
+    let remainder = (value % 2).toString();
+    let newVal = Number.parseInt(value / 2, 0);
```
Comment (bondz) [inline src/binary-reversal/index.js:11]: These can be `const` too :)

File: src/binary-reversal/index.js [hunk 2]
```diff
@@ -3,6 +3,20 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function convertToReveredBinary(value) {
```
Comment (bondz) [inline src/binary-reversal/index.js:6]: Do you mean `reversed`? Sure do like some `revered` binary myself.

Comment (obrienu) [reply src/binary-reversal/index.js:6]: Yes, I meant reversed. A typographic error :)

File: src/binary-reversal/index.js (hunk 1 above)
Comment (obrienu) [reply src/binary-reversal/index.js:11]: Noted, will enforce the use of const in my codes

**Pull request merged**